services/frontend/src/scenes/Profile/Profile.scss

```diff
@@ -86,6 +86,21 @@ $project-card-width: $default-spacing * 45;
         display: flex;
         flex-wrap: wrap;
     }
+
+    .profile-card-edit-skills-button {
+            padding: 3px;

```

Comment:
Similar to the fix I put into CDHSH-89, this should have a larger ripple effect.

In my opinion the actual button itself should be moved inside the skills card on the web page as well, it currently looks out of place. That said, I believe your version was in the mock-ups so maybe it requires more discussion.